Src/DependencyCollector/NuGet/Package.nuspec

```diff
@@ -23,12 +23,13 @@
       <group targetFramework="net45">
         <dependency id="Microsoft.ApplicationInsights" version="[$coresdkversion$]" />
         <dependency id="Microsoft.ApplicationInsights.Agent.Intercept" version="2.0.7" />
+        <dependency id="System.Diagnostics.DiagnosticSource" version="4.4.0-preview1-25210-01" />
```

Comment:
We are about to start releasing 2.4 Beta 1 in couple days - is this package present on the nuget servers reachable by customers? I looked at nuget.org and the last version is 4.3.0, however, this one might not just be visible.